cmsplugin_articles_ai/templates/cmsplugin_articles_ai/category_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "cmsplugin_articles_ai/base.html" %}
+
+{% block page_title %}
+    {{ page_title }}
+{% endblock page_title %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-9">
+            <h2>{{ page_title }}</h2>
+
+            {% include "cmsplugin_articles_ai/paginated_article_list.html" with articles=articles %}
+        </div>
+
+        <div class="col-md-3">
+            <h3>Categories</h3>
+            {% include "cmsplugin_articles_ai/category_list.html" with categories=all_categories %}
+
+            <h3>Tags</h3>
+            {% include "cmsplugin_articles_ai/tag_list.html" with tags=all_tags %}
+        </div>
+    </div>
+{% endblock content %}
```

Comment:
Shouldn't this file be called just a generic `article_list.html` I don't see anything category specific in this.